include/bringauto/fleet_protocol/http_client/FleetApiClient.hpp

```diff
@@ -22,6 +22,14 @@ class FleetApiClient {
 		STATUS_ERROR
 	};
 
+	/**
+	 * @brief Enum class for return codes
+	 */
+	enum class ReturnCode {
+		OK,
+		DELAYED
```

Comment:
What does it mean delayed? If someone reads the doc it is not clear what doe it means.

Reply:
added an explanation